vertica-kafka-scheduler/README.md

```diff
@@ -1,58 +1,303 @@
+
+The scheduler component string must first define the `scheduler` itself, and then add each additional required component with the `--add` option. Each component is separated by a semi-colon. You must pass the `--conf /etc/vkconfig.conf` option to each component definition to provide environment settings.
+
+The following command provides an example format:
+
+```bash
+$ docker run \
+    -v <local-config.conf>:/etc/vkconfig.conf \
+    vertica/kafka-scheduler:<version> bash -c "
+        vkconfig scheduler \ 
+          -- conf /etc/vkconfig.conf \
+          <scheduler-options> ...; \
+        vkconfig <component-1> --add \
+          -- conf /etc/vkconfig.conf \
+          <component-1-options> ...; \
+        vkconfig <component-2> --add \
+          -- conf /etc/vkconfig.conf \
+          <component-2-options> ...; \
+        ...
+        "
+```
+
+For a complete example, see the **Set up Scheduler** section in the [example.sh](example.sh) script in this repository. The following snippet from that section defines the first `microbatch` component:
+
+```bash
+$ docker run \
+...
+vkconfig microbatch --add \  
+  --conf /etc/vkconfig.conf \
+  --microbatch KafkaBatch1 \ 
+  --add-source KafkaTopic1 \ 
+  --add-source-cluster KafkaCluster \
+  --target-schema public \   
+  --target-table KafkaFlex \ 
+  --rejection-schema public \
+  --rejection-table KafkaFlex_rej \
+  --load-spec KafkaSpec; \
+...
 ```
-# Commands to configure a scheduler
-docker run vertica/kafka-scheduler vkconfig scheduler --help
-docker run vertica/kafka-scheduler vkconfig cluster --help
-docker run vertica/kafka-scheduler vkconfig source --help
-docker run vertica/kafka-scheduler vkconfig target --help
-docker run vertica/kafka-scheduler vkconfig load-spec --help
-docker run vertica/kafka-scheduler vkconfig microbatch--help
-docker run vertica/kafka-scheduler vkconfig statistics --help
-
-# Launching the scheduler with some useful docker options
-# use "-v" to map the optional conf file to the container's /etc/vconfig.conf
-# use "-v" to map the log config file to the container's /opt/vertica/packages/kafka/config/vkafka-log-config.xml
-# use "-v" to map the writable log directory to the internal /opt/vertica/log
-# use "--user" to run with the credentials of the log directory
-docker run -it \
+
+### Launch a scheduler
+
+After you [create a scheduler](#create-a-scheduler), launch the scheduler to begin scheduling microbatches. To launch a scheduler, execute a `docker run` command that does the following:
+- Mounts a configuration file as a volume.
+- Specifies the scheduler image name.
+- Mounts the local `vkafka-log-config-debug.xml` file in the container's `/opt/vertica/packages/kafka/config` directory. This file configures log messages to help troubleshoot scheduler issues.
+- Mounts the local `/log` directory in the container's `/opt/vertica/log` directory to write logs to help troubleshoot scheduler issues.
```

Comment:
"mounts `log` from the current directory into the container's `/opt/vertica/log` so that log messages will be written into that local `log` directory."

Reply:
Updated.